platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/widgets/MyLocationView.java

```diff
@@ -546,14 +547,19 @@ public void setContentPadding(int[] padding) {
     contentPaddingY = (padding[1] - padding[3]) / 2;
   }
 
-  private static class GpsLocationListener implements LocationListener {
+  private static class GpsLocationListener implements LocationEngineListener {
 
     private WeakReference<MyLocationView> userLocationView;
 
     GpsLocationListener(MyLocationView myLocationView) {
       userLocationView = new WeakReference<>(myLocationView);
     }
 
+    @Override
+    public void onConnected() {
+      // Nothing
```

Comment:
@tobrun Good thinking, fixed that now with https://github.com/mapbox/mapbox-gl-native/pull/7892/commits/185bbfae4b8819f4a1947c2aff7f21f81ce5d517.